README.md

```diff
@@ -68,10 +68,10 @@ flawz --feeds 2024 --query xz
 
 ### Cargo
 
-**flawz** can be installed from [crates.io](https://crates.io/crates/flawz) using [`cargo`](https://doc.rust-lang.org/cargo/) if [Rust](https://www.rust-lang.org/tools/install) is installed.
+**flawz** can be installed from [crates.io](https://crates.io/crates/flawz) using [`cargo`](https://doc.rust-lang.org/cargo/) if [Rust](https://www.rust-lang.org/tools/install) and SQLite 3 development files (e.g. package `libsqlite3-dev` on Debian an derivatives) are installed.
```

Comment:
I would prefer we give out the full command for installation of SQLite files, e.g. `apt install libsqlite3-dev`

We should probably do that after giving out the `cargo install` command though. Maybe in a GitHub admonition such as:

> [!NOTE] 
> You need to install SQLite 3 development files on Debian and derivates with the following command:
> ```
> apt install libsqlite3-dev
> ```

Reply:
Ok, makes sense.
So I'd change like below.

**flawz** can be installed from [crates.io](https://crates.io/crates/flawz) using [`cargo`](https://doc.rust-lang.org/cargo/) if [Rust](https://www.rust-lang.org/tools/install) is installed.

```sh
cargo install --locked flawz
```

The minimum supported Rust version (MSRV) is `1.74.1`.

> [!NOTE]  
> You need to have SQLite 3 development files installed. On Debian and its derivates you can do so with the following command:
>
> ```sh
> sudo apt install libsqlite3-dev
> ```

Reply:
Looks good! Can you incorporate those changes in this PR? 😇

Reply:
done! 🙂